l10n_it_edi_extension/__init__.py

```diff
@@ -0,0 +1,1188 @@
+# Copyright 2025 Giuseppe Borruso - Dinamiche Aziendali srl
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import controllers
+from . import models
+from . import wizards
+
+import os
+import base64
+from odoo.tools import config
+
+from openupgradelib import openupgrade
+from psycopg2 import sql
+
+from odoo.addons.base.models.ir_qweb_fields import Markup, nl2br, nl2br_enclose
+
+OLD_MODULES = [
+    "l10n_it_fatturapa",
+    "l10n_it_fatturapa_in",
+    "l10n_it_fatturapa_out",
+    "l10n_it_fatturapa_sale",
+    "l10n_it_fiscal_payment_term",
+    "l10n_it_fiscalcode",
+    "l10n_it_ipa",
+    "l10n_it_pec",
+    "l10n_it_rea",
+    "l10n_it_vat_payability",
```

Comment:
Un gran lavoro! :clap:
Purtroppo non posso provarlo perché OpenUpgrade ancora non ha migrato gran che alla `18.0`.

Se questo però è il modulo che dovrebbe coprire le funzionalità di questi moduli, mi aspetto che ci siano dei test.
Non dico tutti quelli dei moduli elencati, che potrebbero facilmente essere un centinaio, però abbiamo fatto un grande lavoro sui test nella `16.0` e non mi va che vada tutto perso.

Reply:
per i test in realtà fa tutto `l10n_it_edi` posso fare solo test per preview dell'xml e la popolazione delle tabella di dettaglio della fattura elettronica

Reply:
Ho preso il primo test di uno dei moduli elencati: https://github.com/OCA/l10n-italy/blob/56621cb667a6d9ed581a55bd452508b28d49e3da/l10n_it_fiscalcode/tests/test_fiscalcode.py#L17-L34

`l10n_it_edi` come fa il calcolo del codice fiscale di un partner?

Per sapere se tutte le funzionalità dei nostri vecchi moduli ci sono in `l10n_it_edi` c'è un modo semplice: eseguire i test automatici.